apps/video/view.html

```diff
@@ -16,9 +16,11 @@
   <body role="application" class="skin-dark layout-fullscreen-player view-activity theme-media">
     <!-- display this at startup while we create thumbnails -->
     <div id="spinner-overlay"><div id="spinner"></div></div>
+    <div id="spinner-overlay" role="progress" data-l10n-id="spinner-overlay">
+      <div id="spinner"></div></div>
     <section role="region" id="player-view">
       <div id="video-container">
-        <video id="player" preload="metadata"></video>
+        <video id="player" preload="metadata" aria-hidden="true"></video>
```

Comment:
So the video element is used to display the video itself, but it makes little sense for the screen reader user, unless it exposes default browser controls. So in our case here, if we use aria-hidden="true" we hide if from the assistive technology only, but keep it visible for the sighted user.